From 1dbd8f371ebc68ea77b91c8618488e3a0360245b Mon Sep 17 00:00:00 2001
From: Alexandr Savca <alexandrsavca89@gmail.com>
Date: Sat, 10 Oct 2020 12:39:08 +0300
Subject: [PATCH] print action message instead of url like something is
 downloading

---
 holehe/core.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/holehe/core.py b/holehe/core.py
index d0821a3..68def1d 100644
--- a/holehe/core.py
+++ b/holehe/core.py
@@ -1081,7 +1081,7 @@ def eventbrite(email):
 
 
 def main():
-    print('Github : https://github.com/megadose/holehe')
+    print('Processing...')
     start_time = time.time()
     parser = argparse.ArgumentParser()
     requiredNamed = parser.add_argument_group('required named arguments')
-- 
2.28.0

