From 6f66b2f5b9de00972b5e483c6f3b2b3033de25a9 Mon Sep 17 00:00:00 2001
From: Funtastix <funtastix@users.noreply.github.com>
Date: Tue, 11 Aug 2020 12:22:48 +1000
Subject: [PATCH 1/5] Update Makefile

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 5d34ed6..2a17e76 100644
--- a/Makefile
+++ b/Makefile
@@ -10,6 +10,7 @@
 LDLIBS		= -lpcap -lcrypt
 CFLAGS		= -pipe -Wall -D_LINUX -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -I../../..
 CFLAGS		+= -D_OPENSSL_MD4
+CFLAGS          += -I/usr/include/openssl
 LDLIBS		+= -lcrypto
 CFLAGS		+= -g3 -ggdb -g
 PROGOBJ		= asleap.o genkeys.o utils.o common.o sha1.o
-- 
2.28.0

