From ac1b82865ff37e395f9be0b20a542172a671f253 Mon Sep 17 00:00:00 2001
From: Funtastix <funtastix@users.noreply.github.com>
Date: Tue, 11 Aug 2020 12:27:56 +1000
Subject: [PATCH 2/5] Update utils.c

---
 utils.c | 22 ++++++++++++----------
 1 file changed, 12 insertions(+), 10 deletions(-)

diff --git a/utils.c b/utils.c
index 4941514..b541047 100644
--- a/utils.c
+++ b/utils.c
@@ -167,20 +167,22 @@ void Collapse(unsigned char *in, unsigned char *out)
 	}
 }
 
-void DesEncrypt(unsigned char *clear, unsigned char *key, unsigned char *cipher)
-{
-	unsigned char des_key[8];
-	unsigned char crypt_key[66];
-	unsigned char des_input[66];
+static DES_key_schedule	key_schedule;
 
+bool DesSetkey(u_char *key)
+{
+	DES_cblock des_key;
 	MakeKey(key, des_key);
+	DES_set_key(&des_key, &key_schedule);
+	return (1);
+}
 
-	Expand(des_key, crypt_key);
-	setkey((char *)crypt_key);
+void DesEncrypt(unsigned char *clear, unsigned char *key, unsigned char *cipher)
+{
+	DesSetKey(key);
 
-	Expand(clear, des_input);
-	encrypt((char *)des_input, 0);
-	Collapse(des_input, cipher);
+	DES_ecb_encrypt((DES_cblock *)clear, (DES_cblock *)cipher,
+	    &key_schedule, 1);
 }
 
 int IsBlank(char *s)
-- 
2.28.0

