From e0f46403ebca23ad90c1d8cad2e00972726bbc23 Mon Sep 17 00:00:00 2001
From: Funtastix <funtastix@users.noreply.github.com>
Date: Tue, 11 Aug 2020 12:34:02 +1000
Subject: [PATCH 4/5] Update utils.h

---
 utils.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/utils.h b/utils.h
index d777c0c..775eff9 100644
--- a/utils.h
+++ b/utils.h
@@ -22,6 +22,8 @@
  * lamont_dump from nmap's utils.cc.  Thanks Fyodor.
  */
 
+#include <des.h>
+
 /* Prototypes */
 void lamont_hdump(unsigned char *bp, unsigned int length);
 unsigned char Get7Bits(unsigned char *input, int startBit);
-- 
2.28.0

