From f545aeb20ecfb6a3cfd9ec160286baebfd97e0e7 Mon Sep 17 00:00:00 2001
From: Alexandr Savca <drop@chinarulezzz.fun>
Date: Sat, 10 Oct 2020 12:11:45 +0300
Subject: [PATCH] use html5lib parser by default

---
 mechanize/_html.py | 16 +++++-----------
 1 file changed, 5 insertions(+), 11 deletions(-)

diff --git a/mechanize/_html.py b/mechanize/_html.py
index a50a17f..cc74f98 100644
--- a/mechanize/_html.py
+++ b/mechanize/_html.py
@@ -185,17 +185,11 @@ def content_parser(data,
     '''
     if not is_html:
         return
-    try:
-        from html5_parser import parse
-    except Exception:
-        from html5lib import parse
-        kw = {'namespaceHTMLElements': False}
-        if transport_encoding and isinstance(data, bytes):
-            kw['transport_encoding'] = transport_encoding
-        return parse(data, **kw)
-    else:
-        return parse(data, transport_encoding=transport_encoding)
-
+    from html5lib import parse
+    kw = {'namespaceHTMLElements': False}
+    if transport_encoding and isinstance(data, bytes):
+        kw['transport_encoding'] = transport_encoding
+    return parse(data, **kw)
 
 def get_title(root):
     for title in root.iter('title'):
-- 
2.28.0

